be/src/exec/arrow/orc_reader.cpp

```diff
@@ -52,6 +55,31 @@ Status ORCReaderWrap::init_reader(const TupleDescriptor* tuple_desc,
         return Status::EndOfFile("Empty Orc File");
     }
 
+    int64_t row_number = 0;
+    int end_group = _total_groups;
+    for (int i = 0; i < _total_groups; i++) {
+        int64_t _offset = _reader->GetRawORCReader()->getStripe(i)->getOffset();
+        int64_t row = _reader->GetRawORCReader()->getStripe(i)->getNumberOfRows();
+        if (_offset < _range_start_offset) {
+            row_number += row;
+        } else if (_offset == _range_start_offset) {
+            _current_group = i;
```

Comment:
`_current_group` may not be set here?
And it may always be 0?

Reply:
In my test for orc file , `_range_start_offset` is one of the `_offset` list